.github/CODEOWNERS

```diff
@@ -6,6 +6,7 @@
 /*/manifest.json                         @DataDog/documentation @DataDog/agent-integrations
 
 # Community partners
+/1e/                                     support@1e.com
```

Comment:
Please add DataDog/marketplace-review to the CODEOWNERS file

Reply:
Added